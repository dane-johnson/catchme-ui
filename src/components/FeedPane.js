import React from 'react'
import { Well } from 'react-bootstrap'

const style = {
  listStyle: 'none'
}

const FeedPane = ({feed}) => {
  const items = feed.map((item, key) => <li key={key}>{item}</li>)
  return (
    <Well>
      <ul style={style}>
        {items}
      </ul>
    </Well>
  )
}

export default FeedPane
