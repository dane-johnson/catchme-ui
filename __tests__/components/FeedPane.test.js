import React from 'react'
import renderer from 'react-test-renderer'
import FeedPane from '../../src/components/FeedPane'

const mockFeed = [
  'Airplane killed Donnie',
  'Donnie killed Frank'
]

describe('<FeedPane />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <FeedPane feed={mockFeed} />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
