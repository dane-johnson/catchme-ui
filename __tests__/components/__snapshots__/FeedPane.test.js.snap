exports[`<FeedPane /> renders correctly 1`] = `
<div
  className="well">
  <ul
    style={
      Object {
        "listStyle": "none",
      }
    }>
    <li>
      Airplane killed Donnie
    </li>
    <li>
      Donnie killed Frank
    </li>
  </ul>
</div>
`;
